.env.example

```diff
@@ -13,19 +13,19 @@ BOSON_TOKEN=0x056b1ce5510ceefa6c33600551e55333aff8296f
 # DAI token Rinkeby address
```

Comment:
This should be without rinkeby.

Also line 10 `# local hardhat address #9` is not ok description.

Reply:
@zajck not sure what you mean, re: rinkeby ? what DAI token address is that ? which chain ?

And i can update line 10

Reply:
I've discussed that with @hswopeams 

The values are ok, it just wasn't clear for me, if the defaults have any meaning. And they might confuse someone when they see them first.

So I would just add comment like "Default values for BOSON_TOKEN and DAI_TOKEN are required for local deployment. To do deployment on chain, adjust them accordingly" (or something similar that tells the same)

Reply:
@zajck updated.